=========================================================================================================================

# How to determine the age of the Universe in a Jupyter Notebook

=========================================================================================================================

## 10 Methods 

=========================================================================================================================

List of physical (NIST CODATA2018) and mathematical constants used:
* $G=6.6743.10^{-11}$ $m^{3}.kg^{-1}.s^{-2}$ CODATA2018
* $c=299792458.0$ m/s CODATA2018
* $h=6.62607015e-34$ $J.Hz^{-1}$ CODATA2018
* $\hbar=1.0545718176461565e-34$ $J.s$ CODATA2018
* ƛ$_e$ $=3.861 592 6796.10^{-13}$m Reduced (Electron) Compton Wavelength CODATA2018
* $Euler Mascheroni=0.5772156649015329$
* Atiyah's $\Gamma=25.178097241906$
* Inverse of the fine structure constant $a=137.0359990836958$ also know as the inversed electromagnetic coupling constant
* Mass of the electron $m_e=9.1093837015.10^{-31}$ kg CODATA2018
* $G=6.6755.10^{-11}$ $m^{3}.kg^{-1}.s^{-2}$ by T.Quinn et al. (2013) BIPM Sevres
* $G=6.675453818.10^{-11}$ $m^{3}.kg^{-1}.s^{-2}$ by Francis M. sanchez

In [132]:
from scipy import constants
import numpy as np

In [133]:
constants.G

6.6743e-11

In [134]:
constants.c

299792458.0

In [135]:
constants.h

6.62607015e-34

In [136]:
constants.hbar

1.0545718176461565e-34

In [137]:
np.euler_gamma

0.5772156649015329

In [138]:
constants.m_e

9.1093837015e-31

=========================================================================================================================

# Universe Horizon Radius calculus in 3 minutes (method 1)
$$R_U=\frac{2.\hbar^2}{G.m_e.m_p.m_n}$$

In [139]:
R_U=2*constants.hbar**2/(constants.G*constants.m_e*constants.m_p*constants.m_n)
print(R_U)

1.3058515261319119e+26


## One obtains 13.802861522337478 Gly with method 1

In [140]:
R_U/(constants.light_year*10**9)

13.802861522337478

=========================================================================================================================

# Nambu Universe Radius calculus in 3 minutes (method 2)
$$R_N=\frac{2.\hbar^2}{G.(m_N)^3}$$

### https://en.wikipedia.org/wiki/Yoichiro_Nambu

With Nambu mass $$m_N=m_e.a$$ and $$a=1/\alpha$$

In [141]:
a=(1/constants.fine_structure)
print(a)

137.0359990836958


In [142]:
m_N=constants.m_e*a
print(m_N)

1.2483134965717876e-28


In [143]:
R_N=2*constants.hbar**2/(constants.G*((a)*constants.m_e)**3)
print(R_N)

1.7131902573578377e+26


## One obtains 18.108435308700802 Gly with method 2

In [144]:
R_N/(constants.light_year*10**9)

18.108435308700802

$$R_N'=ƛ_e(2R/R_N)^{210}$$ and $$R=\frac{2.\hbar^2}{G.m_e.m_p.m_H}$$

$$H=p+1-\frac{1/2}{a'^2}$$ and $a'=H/p=137.11$

In [145]:
H=constants.m_e*1837.152645
print(H)

1.6735328361530618e-27


In [146]:
R=2*constants.hbar**2/(constants.G*constants.m_e*constants.m_p*H)
print(R)

1.3069397756805032e+26


In [147]:
R_Nprime=(constants.hbar/(constants.m_e*constants.c))*(2*R/R_N)**210
print(R_Nprime)

1.3108749648749215e+26


## One obtains 13.855959311747807 Gly with method 2'

In [148]:
R_Nprime/(constants.light_year*10**9)

13.855959311747807

=========================================================================================================================

# Eddington Universe Radius calculus in 3 minutes (method 3)
$$R_{Edd}=\frac{2.\hbar^2}{G.m_e.m_p^2}$$

### https://en.wikipedia.org/wiki/Eddington_number

In [149]:
# Number of Hydrogen Atoms of mass m_H in the visible universe
N_Edd=136*2**256
print(N_Edd)

15747724136275002577605653961181555468044717914527116709366231425076185631031296


In [150]:
m_H=1837.152645*constants.m_e
print(m_H)

1.6735328361530618e-27


In [151]:
20/3

6.666666666666667

In [152]:
R_Edd=(6.666666666666667*N_Edd*constants.G*m_H)/constants.c**2

## One obtains 13.791148189117836 Gly for R_Edd

In [153]:
R_Edd/(constants.light_year*10**9)

13.791148189117836

In [154]:
R_Edd2=2*constants.hbar**2/(constants.G*constants.m_e*constants.m_p**2)
print(R_Edd)

1.3047433592466462e+26


## One obtains 13.82188765312757 Gly for R_Edd2

In [155]:
R_Edd2/(constants.light_year*10**9)

13.82188765312757

=========================================================================================================================

# Francis M. Sanchez Universe Radius calculus in 3 minutes (method 4)
$$R_{Francis.M.Sanchez}=ƛ_e.g(6)$$

With $$g(k)=exp(2^{k+1/2})/k$$ when $$k=6$$

And $$ƛ_e=\frac{\hbar}{m_e.c}$$ as the reduced Compton electron wavelength radius

In [156]:
# Reduced Compton Wavelength
(constants.hbar/(constants.m_e*constants.c))

3.8615926796089057e-13

In [157]:
import numpy as np
from scipy import constants
R_FMS=(constants.hbar/(constants.m_e*constants.c))*np.exp(2**(6+(1/2)))/6
print(R_FMS)

1.307568892158173e+26


## One obtains 13.821014095558315 Gly with method 4

In [158]:
R_FMS/(constants.light_year*10**9)

13.821014095558315

=========================================================================================================================

# Universe Radius calculus based on Rydberg reduced wavelength (method 5) 
$$R_{Rydberg}=\frac{(ƛ_{Ryd}.n^4)^2}{ƛ_p}$$

In [159]:
a=constants.fine_structure**-1
print(a)

137.0359990836958


In [160]:
a**2

18778.865044866678

In [161]:
# n equals mass ratio between Neutron and Electron
n=(constants.m_n)/(constants.m_e)
print(n)

1838.6836617324586


In [162]:
n**4

11429522003521.688

In [163]:
lambdabarRyd=2*(constants.hbar/(constants.m_e*constants.c))*(a**2)
print(lambdabarRyd)

1.4503265557724145e-08


In [164]:
lambdabar_p=(constants.hbar/(constants.m_p*constants.c))
print(lambdabar_p)

2.1030891033555923e-16


In [165]:
R_Ryd=((lambdabarRyd*n**4)**2)/lambdabar_p
print(R_Ryd)

1.3065621143319103e+26


## One obtains 13.810372445537942 Gly with method 5

In [166]:
# Incorrect result , method to be modified
R_Ryd/(constants.light_year*10**9)

13.810372445537942

=========================================================================================================================

# Universe Radius calculus based on pi (method 6) 
$$R_{\pi}=ƛ_e.\pi^{155/2}$$

In [167]:
R_pi=(constants.hbar/(constants.m_e*constants.c))*constants.pi**(155/2)
print(R_pi)

1.3058148293550039e+26


## One obtains 13.802473637099501 Gly with method 6

In [168]:
R_pi/(constants.light_year*10**9)

13.802473637099501

=========================================================================================================================

# Universe Radius calculus based on 2 (method 7) 
$$R_{2}=ƛ_e.2^{128}$$

In [169]:
R_2=(constants.hbar/(constants.m_e*constants.c))*2**128
print(R_2)

1.3140318971018876e+26


## One obtains 13.889328111716429 Gly with method 7

In [170]:
R_2/(constants.light_year*10**9)

13.889328111716429

=========================================================================================================================

# Universe Radius calculus based on Fermi's mass and Kotov period (method 8) 

$$R_{kotov-fermi}=\frac{2.(c.t_k)^2/ƛ_e}{F^2}$$

Fermi's mass given by the relation: $$F=(2\times 137 \times \Gamma)^{3/2}$$

In [171]:
# Atiyah's Gamma
Gamma=a*np.euler_gamma/np.pi
print(Gamma)

25.178097241906


In [172]:
# F**(2/3)=2*137*Gamma
# Fermi's mass given by 
F=(2*137*Gamma)**1.5
print(F)

573007.364905975


In [173]:
((2*(constants.c*9600.6)**2)/(constants.hbar/(constants.m_e*constants.c)))/F**2

1.3067162206121416e+26

In [174]:
R_kf=(2*(constants.c*9600.6)**2)/(constants.hbar/(constants.m_e*constants.c)) / F**2
print(R_kf)

1.3067162206121416e+26


https://en.wikipedia.org/wiki/Light-year

In [175]:
# https://en.wikipedia.org/wiki/Light-year
constants.light_year

9460730472580800.0

In [176]:
R_kf/(constants.light_year*10**9)

13.812001350205271

=========================================================================================================================

# Universe Radius calculus based on "a"  (method 9) 
$$a=1/\alpha$$ $\alpha$ being the fine structure constant  $$R_{a}=ƛ_e.(2.\pi^2.a^3)^5$$

In [177]:
lambdabare=(constants.hbar/(constants.m_e*constants.c))
print(lambdabare)

3.8615926796089057e-13


In [178]:
(2*constants.pi**2)*((1/constants.fine_structure)**3)**5

2.2276645699260076e+33

In [179]:
R_a=((constants.pi**2*(2*(1/constants.fine_structure)**3))**5)*(constants.hbar/(constants.m_e*constants.c))
print(R_a)

1.3059760854303987e+26


In [180]:
R_a/(constants.light_year*10**9)

13.80417811516134

=========================================================================================================================

# Universe Radius calculus based on $c$ and $M_U$ (method 10) 


$$R_U = \frac{2\cdot G\cdot M_U}{c^2}$$

$$M_U= \frac{R.c^2}{2G}$$

In [181]:
# Use m_H instead of m_n
M_U=(13.81197677*constants.c**2)/2*constants.G
print(M_U)

41425997.35400031


$$M_U = \frac{({ħc}{G})^2}{m_em_pm_h}$$

In [182]:
M_U=(((constants.Planck*2*constants.pi)*constants.G*constants.c)**2)/(constants.m_e*m_H*constants.m_p)
print(M_U)

2721470494531313.5


In [183]:
R_method10=2*constants.G*M_U/constants.c**2
print(R_method10)

4.042015212013433e-12


In [184]:
# Incorrect result , method to be modified
R_method10/(constants.light_year*10**9)

4.272413450238382e-37

=========================================================================================================================

# Universe Radius calculus based on $ƛ_w$, $\gamma$ and $\Gamma$ (method 11) 

$$R_{ƛ_w}=ƛ_w.(2^{70}.\gamma.n^3/\Gamma^4)^2$$

$$ƛ_w=ƛ_e/F^2$$

In [185]:
import numpy as np

In [186]:
a=constants.fine_structure**-1
print(a)

137.0359990836958


In [187]:
# Atiyah's Gamma
Gamma=a*np.euler_gamma/np.pi
print(Gamma)

25.178097241906


In [188]:
np.pi

3.141592653589793

In [189]:
ƛ_w=(constants.hbar/(constants.m_e*constants.c))/F**2
print(ƛ_w)

1.1761048867371158e-24


In [190]:
R_w=(((2**70*np.euler_gamma*n**3)/Gamma**4)**2)*ƛ_w
print(R_w)

1.306713911982143e+26


In [191]:
R_w/(constants.light_year*10**9)

13.811976947966931

=========================================================================================================================

# Measurement Confirmation of the methods given above by:

### Confirmation Measurement #1
Universe Radius measurement based on supernovae 1a
$$R_{S1a}=13.6(6)Gly$$

### Confirmation Measurement #2 (Planck Space Observatory)
https://en.wikipedia.org/wiki/Planck_(spacecraft)
13.8 billion years (as of 2015) – 13.799±0.021 billion (109) years within the Lambda-CDM concordance model.

$$R_{Planck}=13.799±0.021 Gly$$

### Confirmation Measurement #3 (James Webb Space Telescope)

#### https://en.wikipedia.org/wiki/James_Webb_Space_Telescope

Measurements due to start in 2021

=========================================================================================================================

# Other estimates

## Universe mass calculus given by the relation:

In [192]:
%%latex
\begin{align}
\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M \\
\end{align}

<IPython.core.display.Latex object>

In the following example, one can write the equation with Latex:

In [193]:
from IPython.display import Latex
Latex(r"""\begin{eqnarray}
\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M  \\
\ R_U = \frac{2\cdot G\cdot M_U}{c^2} \\
\ M_U = \frac{({ħc}{G})^2}{m_em_pm_h} \\
\frac{r_H}{ƛ_e} = \frac{\sum\frac{1}{n}}{\sum\frac{1}{n^2}}\\
\end{eqnarray}""")


<IPython.core.display.Latex object>

=========================================================================================================================

:star:

# Docs

This project aims at providing a repository dedicated to Dr Francis Michel Sanchez's work in holography and cosmology; it is just a mere continuation of what was initiated and published in a set of theoritical physics articles.


## Publications
* Please, cite this paper (http://vixra.org/pdf/1811.0146v8.pdf) if you employ this package in your research work.
* [Vixra Publications](http://vixra.org/author/francis_m_sanchez)
* [Rxiv Publications](http://rxiv.org/author/f_m_sanchez)
* [Nature: Observations of solar oscillations with periods of 160 minutes](https://www.nature.com/articles/277635a0)
* [A coherent resonant cosmology approach and its implications in microphysics and Biophysics Pages 375-407 of Quantum Systems in Physics, Chemistry](https://www.springer.com/fr/book/9783319502540?token=gbgen&wt_mc=GoogleBooks.GoogleBooks.3.EN#)
* [From Coherent Cosmic Oscillations to the Steady-State Cyclic Ultrafast](https://rxiv.org/abs/1401.0228)
* [Evidence for a Steady-State, Holographic, Tachyonic and Super-Symmetric Cosmology](https://www.researchgate.net/publication/267430152_Evidence_for_a_Steady-State_Holographic_Tachyonic_and_Super-Symmetric_Cosmology)
* [academia.edu Correcting Five Historical Faults in Modern Cosmology](https://www.academia.edu/30843365/Correcting_Five_Historical_Faults_in_Modern_Cosmology)

## Videos

* [L'enigme des grands nombres Part1](https://www.youtube.com/watch?v=ev_eIqLtK8s)
* [L'enigme des grands nombres Part2](https://www.youtube.com/watch?v=i4w392nYqns)
* [L'enigme des grands nombres Part3](https://www.youtube.com/watch?v=b4WvI0yVXnE)
* [Exposés de Francis Sanchez playlist created by J.Bailhache](https://www.youtube.com/playlist?list=PLQXUNrzoAeFbmfHo3jgWzzExlVoBn2JUw)
* [Khelif/Torgoz/Sanchez' Debate on LE FORMALISME EN PHYSIQUE ET EN SCIENCE](https://www.youtube.com/watch?v=TEK2hCI2we0)

* [La science du 3eme millenaire playlist created by J.Bailhache](https://www.youtube.com/playlist?list=PLQXUNrzoAeFbLOuEsltAUWJWCcAEgeGJE)
* [Francis Sanchez Cosmologie 23/03/2017](https://www.youtube.com/watch?v=NmvvIQjxBw8)
* [La complexité mathématique face à la simplicité cosmique - Exposé de Francis Sanchez](https://www.youtube.com/watch?v=dyUlUahgHGg)
* [La méthode scientifique en question et les constantes fondamentales - Exposé](https://www.youtube.com/watch?v=zUt1hxTdiuA)
* [Jean Maruani chez lui et Francis Sanchez autour du nombre 237 29. 12. 17](https://www.youtube.com/watch?v=05g416VWjnw)
* [Pr. Michel Grosmann: Holographie quanto-cosmique](https://www.youtube.com/watch?v=M7bJnekDIt0)
* [Pr. Renée Veysseyre: La constante d'Atiyah réunifie Physique et Mathématique](https://www.youtube.com/watch?v=mbPpKqakDAU)
* [Christian Bizouard, astronome: Omniprésence astrophysique du cycle de Kotov](https://www.youtube.com/watch?v=DGeBC73kKvQ)
* [Pr. Francis Sanchez: Historique, Principe Calculatoire, Axe Topologique et Principe Holique](https://www.youtube.com/watch?v=IgKSrfsqFEs)

## Books

* [Bizouard, Kotov, Sanchez wrote in 20th Natural Philosophy Alliance Proceedings by David de Hilster, published by Lulu.com in 2013]()
* [Quantum Systems in Physics, Chemistry, Springer Ed](https://www.amazon.com/gp/search?index=books&linkCode=qs&keywords=9783319502557)
* [The Worldwide List of Alternative Theories and Critics](https://books.google.com/books?isbn=2902425171)
* [The Dirac Electron: From Quantum Chemistry to Holistic Cosmology by J.Maruani](https://onlinelibrary.wiley.com/doi/abs/10.1002/jccs.201500374#)
* [solar 22 years cycle](https://link.springer.com/article/10.1007%2Fs10509-016-2985-8)
* [Current Issues in Cosmology](https://www.cambridge.org/gb/academic/subjects/physics/cosmology-relativity-and-gravitation/current-issues-cosmology?format=HB&isbn=9780521858984)
* ```@inproceedings{Sanchez2015RemarkablePO,
  title=Remarkable Properties of the Eddington Number 137 and Electric Parameter 137 . 036 excluding the Multiverse Hypothesis},
  author={Francis M. Sanchez and Valery A. Kotov,
  year=2015```

## Papers

* [College de France (2004): Discussions sur les oscillations cosmiques, les nombres sans dimension et les periodicites en microphysique et cosmologie - College de France 27th Feb 2004](https://lempel.pagesperso-orange.fr/occ_cdf.pdf)

More to come soon stay tuned.../.:stars:.

=========================================================================================================================

In [194]:
#Write your equation in a dedicated cell with latex

from IPython.display import Latex
Latex(r"""\begin{eqnarray}
\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M \\
\frac{\pi_Pt}{\pi_Bb} = 6 \cdot \frac{\pi^5}{p_H^9} \\
\end{eqnarray}""")

<IPython.core.display.Latex object>

In [195]:
#Write Bekenstein-Hawking Entropy 
from IPython.display import Latex
Latex(r"""\begin{eqnarray}
S = \frac{c^3 k A}{4 \hbar G} \\
\end{eqnarray}""")

<IPython.core.display.Latex object>